aries-site/src/pages/components/pagination.mdx

```diff
@@ -1,90 +1,65 @@
-
-## Variants
-
-In addition to [DataTable](/components/datatable#paginated) and [List](/templates/lists#paginated), here is another way pagination can be implemented.
+<Example
+  width="100%"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/ToggleGroupViews.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/CardView.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/ListView.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/utils.js',
+  ]}
+>
+  <ToggleGroupViews />
+</Example>
 
-### Paginated table
+## Anatomy
 
-Tables often contain large amounts of data. To improve the efficiency that a user can navigate to a specific item or range of items, use pagination.
+<Example
+  caption="Diagram illustrating the parts composing Pagination."
+  plain
+>
+  <PaginationAnatomy />
+</Example>
 
-<ExampleImagePreview
-  label="See Paginated Table"
-  href="/components/datatable#paginated"
-  src="/pagination-table-preview.png"
-  alt="HPE Design System Paginated Table Example"
-/>
+| Label | Region          | Purpose                                                                            | Required | Notes                                                                                                                                      |
+| :---: | --------------- | ---------------------------------------------------------------------------------- | :------: | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Container**   | Container, organizes its children, where they appear, how they adapt responsively. | Required | --                                                                                                                                         |
+| **2** | **Summary**     | Provide clarity on what range of items are in view.                                | Optional | If no label is shown, icon is required.                                                                                                    |
+| **3** | **StepOptions** | Displays the current number of items within a page and allows the user to modify.  | Optional | Step defaults to 10. Dropdown options default to 10, 25, 50, 100. The default step and step options can be customized to fit the use case. |
+| **4** | **Navigation**  | Allow user to jump to a specific page.                                             | Required | a11yTitle required. If no icon is shown, label is required.                                                                                |
```

Comment:
I don't understand what "If no icon is shown, label is required." means.

Reply:
removed